.eslintrc

```diff
@@ -6,5 +6,8 @@
       "ignoreUrls": true,
       "ignorePattern": "^\\s*var\\s.+=\\s*(require\\s*\\()|(/)"
     }]
+   },
+   "parserOptions": {
+      "ecmaVersion": 8
```

Comment:
We cannot use `async` functions in generator-loopback yet, we need to keep support for Node.js 6.x.

Please revert this change.

Reply:
That's great. I was going to update all the `this.async()` place to use `async await`. 
Reverting it.

Reply:
I like the idea of getting rid of `this.async()` - if you like to do that then I fully support you in that change!

The trick is to use Promise APIs instead of async functions.

For example:

```ts
async function foo() {
  await bar();
  return 'result';
}
```

needs to be written in Promise-only styler as follows:

```ts
function foo() {
  return bar
    .then(() => 'result');
}
```